content/release-10-17-0/devices-10-17-0.md

```diff
@@ -137,6 +140,22 @@ Build version</th>
 <td>Core</td>
 </tr>
 
+<tr>
+<td>Device Management</td>
+<td>In the SmartREST template editor, issues with the presentation of the <b>External ID type</b> field for Inventory POST messages have been fixed. Under <b>CSV preview</b> the generated "Template creation CSV" has been adjusted to include the ID/externalId/externalIdType values.</td>
```

Comment:
```suggestion
<td>In the SmartREST template editor, issues with the presentation of the <b>External ID type</b> field for Inventory POST messages have been fixed. Under <b>CSV preview</b> the generated "Template creation CSV" has been adjusted to include the ID, externalId and externalIdType values.</td>
```

Reply:
I´m wondering if we have to use any highlighting if we use this format, like "ID", "externalId" and "externalIdType".